README.md

```diff
@@ -1,19 +1,21 @@
 
 <p align="center">
-<img src="http://www.itowns.fr/images/logo-itowns2XS.png" />
+<img src="https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itowns_logo_300x134.png" />
 </p>
 
-# iTowns V2
-
 ## What is it?
 
 iTowns is a [Three.js](https://threejs.org/)-based framework written in Javascript/WebGL for visualizing 3D geospatial data.
+It can connect to WMS/WMTS/TMS servers including elevation data and load many different data formats (3dTiles, gpx, KML and much much more).
+
+iTowns has been redesigned from this [early version](https://github.com/iTowns/itowns-legacy)
 
-It can connect to WMS/WMTS/TMS servers including elevation data and load many different data format (3dTiles, gpx, KML and much much more).
+![iTowns screenshot](https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itownsReleaseXS.jpg)
 
-The V2 of iTowns Open Source is a refactoring of [iTowns V1](https://github.com/iTowns/itowns-legacy).
+## API documentation and demo 
+**[API Documentation](http://www.itowns-project.org/itowns2/API_Doc/)**
+**[Demo](http://www.itowns-project.org/itowns2/examples/globe.html)**
```

Comment:
You have the exact same link in the `examples` section just below. Maybe rename the `Examples` section to `Demo`?